aea/skills/base.py

```diff
@@ -1142,6 +1146,14 @@ def _print_warning_message_for_unused_classes(
             set_of_unused_classes = set(
                 filter(lambda x: x not in used_classes, set_of_classes)
             )
+            # filter out classes that are from other packages
+            set_of_unused_classes = set(
+                filter(
+                    lambda x: not str.startswith(x.__module__, "packages."),
+                    set_of_unused_classes,
+                )
+            )
+
```

Comment:
this is more of a workaround, but does the trick for the `tac_control_contract`skill

Reply:
Hm doesn't that also filter for given skill's classes?

Reply:
no because the loaded modules are not imported with the `packages...` prefix, but only with the suffix relative to the skill package (e.g. `behaviours`, `path.to.module`)